DRCODEOWNERS

```diff
@@ -19,26 +18,26 @@
-/custom_model_runner/datarobot_drum/resource/default_typeschema     @datarobot/core-modeling
-/public_dropin_environments/java_codegen                            @datarobot/custom-models
-/public_dropin_environments/julia_mlj                               @datarobot/custom-models
-/public_dropin_environments/python3_onnx                            @datarobot/custom-models
-/model_templates/*onnx                                              @datarobot/custom-models
-/public_dropin_environments/python3_keras                           @datarobot/custom-models @datarobot/core-modeling
-/public_dropin_environments/python3_pmml                            @datarobot/custom-models @datarobot/core-modeling
-/public_dropin_environments/python3_pytorch                         @datarobot/custom-models @datarobot/core-modeling
-/public_dropin_environments/python3_sklearn                         @datarobot/custom-models @datarobot/core-modeling
-/public_dropin_environments/python3_xgboost                         @datarobot/custom-models @datarobot/core-modeling
-/public_dropin_environments/r_lang                                  @datarobot/custom-models @datarobot/core-modeling
-/task_templates                                                     @datarobot/core-modeling
-/tests/drum                                                         @datarobot/custom-models @datarobot/core-modeling
-/tests/functional/test_custom_task_templates.py                     @datarobot/core-modeling
-/tests/functional/test_drop_in_environments.py                      @datarobot/custom-models @datarobot/core-modeling
-/tests/functional/test_drum_push.py                                 @datarobot/custom-models @datarobot/core-modeling
-/MODEL-METADATA.md                                                  @datarobot/custom-models @datarobot/core-modeling
-/README.md                                                          @datarobot/custom-models @datarobot/core-modeling
-/VALIDATION-SCHEMA.md                                               @datarobot/core-modeling
\ No newline at end of file
+/                                                                 @datarobot/custom-models
+/MODEL-METADATA.md                                                @datarobot/custom-models @datarobot/core-modeling
+/README.md                                                        @datarobot/custom-models @datarobot/core-modeling
+/VALIDATION-SCHEMA.md                                             @datarobot/core-modeling
+/custom_model_runner                                              @datarobot/custom-models @datarobot/core-modeling
+/custom_model_runner/datarobot_drum/drum/custom_fit_wrapper.py    @datarobot/core-modeling
+/custom_model_runner/datarobot_drum/drum/typeschema_validation.py @datarobot/core-modeling
+/custom_model_runner/datarobot_drum/resource/default_typeschema   @datarobot/core-modeling
+/drapps                                                           @datarobot/no-code-apps
```

Comment:
I added only this line. All other changes was requested by test and done by `dr-code-owners` tool.